public/app/features/dashboard/components/HelpWizard/SupportSnapshotService.ts

```diff
@@ -70,17 +72,23 @@ export class SupportSnapshotService extends StateManagerBase<SupportSnapshotStat
   }
 
   async buildDebugDashboard() {
-    const { panel, randomize, snapshotUpdate, iframeLoading, currentTab } = this.state;
+    const { panel, randomize, snapshotUpdate } = this.state;
     const snapshot = await getDebugDashboard(panel, randomize, getTimeSrv().timeRange());
     const snapshotText = JSON.stringify(snapshot, null, 2);
     const markdownText = getGithubMarkdown(panel, snapshotText);
     const snapshotSize = formattedValueToString(getValueFormat('bytes')(snapshotText?.length ?? 0));
 
-    if (iframeLoading && currentTab === SnapshotTab.Support) {
-      setDashboardToFetchFromLocalStorage({ meta: {}, dashboard: snapshot });
+    let scene: SceneObject | undefined = undefined;
+    if (config.featureToggles.scenes && !panel.isAngularPlugin()) {
```

Comment:
This is gated with `scenes` flag for now...  but I think we should gate it as soon as possible.  First real use of scenes in core 🎉

Reply:
I think we should remove this scenes feature toggle gating